OrbitBase/include/OrbitBase/Profiling.h

```diff
@@ -18,9 +29,9 @@ inline void clock_gettime(uint32_t, struct timespec* spec) {
 }
 #endif
 
-inline TickType OrbitTicks(uint32_t a_Clock = 1 /*CLOCK_MONOTONIC*/) {
+inline TickType MonotonicTimestampNs() {
   timespec ts;
-  clock_gettime(a_Clock, &ts);
+  clock_gettime(CLOCK_MONOTONIC, &ts);
   return 1000000000ll * ts.tv_sec + ts.tv_nsec;
 }
 
```

Comment:
maybe inline TickToNanoseconds is it used anywhere? if so change the code to use TickToDuration

The double variant also seems to be not perfect - I think we should use duration where possible.

Add unit-tests for these? The are probably trivial but better to have them.

Reply:
I removed TickToNanoseconds. Again this PR is simply moving code around... I agree that we should have tests and possibly use duration, but this is out of the scope of this PR.